### PR TITLE
Add documentation for ES version config in Shopware 5.5.5

### PR DESCRIPTION
**Pull request by soebbing**

### CODE REVIEW

File: source/sysadmins-guide/elasticsearch-setup/index.md [hunk 1]
```diff
@@ -93,6 +93,24 @@ Shopware 5 communicates with Elasticsearch using the latter's REST API. The `hos
     <strong>Note:</strong> For a single node configuration, which is sufficient for a development environment, it is necessary to configure a `number_of_replicas` of `0`, otherwise the indexing process would wait for cluster health `green`, which can't be reached if no replicas can be applied.
 </div>
 
+### Defining Elasticsearch version to reduce calls
+
+By default, Shopware makes an `info` request to the Elasticsearch backend to be able to determine the the version of Elasticsearch that is being used. In high load environments, this can create unnecessary additional load on all services due to the slight overhead these requests create.
```
Comment (uehler) [inline source/sysadmins-guide/elasticsearch-setup/index.md]: the the version

**Comment by mitelg** [comment]
please talk to @PascalThesing now there are two PRs https://github.com/shopware/devdocs/pull/951

File: source/sysadmins-guide/elasticsearch-setup/index.md (hunk 1 above)
Comment (soebbing) [reply source/sysadmins-guide/elasticsearch-setup/index.md]: Thanks!

**Comment by soebbing** [comment]
> please talk to @PascalThesing now there are two PRs #951

I changed my PR a bit in regard of the other PR. But I think they complement each other

**Pull request merged**